src/main/resources/templates/dashboard.html

```diff
@@ -0,0 +1,222 @@
+				{% include "base/partial/dashboard-card-list-element" with {
+					"imageUrl": "http://www.gantrypark.com/Portals/12/Users/066/14/53314/adam-parker-large.jpg",
+					"title": "PROVIDED FEEDBACK",
+					"subtitle": "Placeholder title for module: language here" } %}
+
+				<li class="date">THURSDAY, MAY 4</li>
+				{% include "base/partial/dashboard-card-list-element" with {
+					"imageUrl": "http://www.gantrypark.com/Portals/12/Users/066/14/53314/adam-parker-large.jpg",
+					"title": "EVALUATED CHALLENGE",
+					"subtitle": "Placeholder title for module: language here and this one is too long for the page so it will cause a line wrap but that's ok." } %}
+				{% include "base/partial/dashboard-card-list-element" with {
+					"imageUrl": "http://www.gantrypark.com/Portals/12/Users/066/14/53314/adam-parker-large.jpg",
+					"title": "PROVIDED FEEDBACK",
+					"subtitle": "Placeholder title for module: language here" } %}
+				{% include "base/partial/dashboard-card-list-element" with {
+					"imageUrl": "http://www.gantrypark.com/Portals/12/Users/066/14/53314/adam-parker-large.jpg",
+					"title": "EVALUATED CHALLENGE",
+					"subtitle": "Placeholder title for module: language here" } %}
+			</ul>
+		</div>
+
+		<div class="col col-4 centered">
+			<div class="title">Your Rating</div>
+			<div class="dashboard-guru-rating">
+				<div class="average">
+					<div class="title">★ 4.89</div>
+					<div class="subtitle">Your average score is comprised of anonymous ratings from
+						users participating in challenges.</div>
+				</div>
+				<div class="tri">
+					<div>
+						<div class="rating">★ 4.79</div>
+						<div class="metric">TIMELINESS</div>
+					</div>
+					<div>
+						<div class="rating">★ 4.98</div>
+						<div class="metric">FEEDBACK QUALITY</div>
+					</div>
+					<div>
+						<div class="rating">★ 4.60</div>
+						<div class="metric">RESPONSE RATE</div>
+					</div>
+				</div>
+			</div>
+		</div>
+	</div>
+
+	<div class="dashboard-row">
+		<div class="col col-6 centered">
+			<div class="title">Skill Development</div>
+			<div class="middle-aligner"></div>
+			<div class="middle-aligned h-scroll">
+				{% include "base/partial/radial-progress" with {
+					"percent": "35",
+					"subtitle": "Java" } %}
+				{% include "base/partial/radial-progress" with {
+					"percent": "90",
+					"subtitle": "Python" } %}
+				{% include "base/partial/radial-progress" with {
+					"percent": "45",
+					"subtitle": "HTML &amp; CSS" } %}
+			</div>
+		</div>
+		<div class="col col-6 dashboard-active-hours centered">
+			<div class="title">Active Hours <span class="hint">This Week</span></div>
+			<div class="middle-aligner"></div>
+			<img class="middle-aligned" src="https://s3-us-west-2.amazonaws.com/jpli-capstone/icons/dashboard-hours.png" />
+		</div>
+	</div>
+
+	<div class="dashboard-row">
+		<div class="col col-8 no-padding">
+			<div class="title">Challenges (30)</div>
+			<table class="dashboard-challenges">
```

Comment:
Are the first and last <th>/<td> elements used just for padding? Can we add padding on the first/last children of each <tr>, or is there a way to pad each <tr>? If it's too much work, it can be left as is, since it's just a nitpicky thing